Source/santasyncservice/SNTSyncStage.mm

```diff
@@ -211,8 +218,19 @@ - (NSData *)dataFromRequest:(NSURLRequest *)request timeout:(NSTimeInterval)time
 - (NSError *)performRequest:(NSURLRequest *)request
                 intoMessage:(google::protobuf::Message *)message
                     timeout:(NSTimeInterval)timeout {
-  NSData *data = [self dataFromRequest:request timeout:timeout];
-  if (data.length == 0) return nil;
+  NSError *error;
+  NSData *data = [self dataFromRequest:request timeout:timeout error:&error];
+  if (*error != NULL) {
```

Comment:
No need to deference the `NSError *`.
`if (error) {`

Reply:
Fixed